### ISSUE TITLE
Unable to convert .lang file that contains empty line

### ISSUE THREAD

**Issue by zomabies** [ichttt/MCLang2Json#4]
It gives an error when trying to convert file that contains empty line.

https://i.imgur.com/SfB8hcT.png  
snippet of en_us.lang from release 1.12.2:
```
gui.no=No
gui.none=None
gui.all=All

gui.recipebook.moreRecipes=Right Click for more
gui.recipebook.toggleRecipes.all=Showing all
```

results:
```json
{
  "gui.no": "No",
  "gui.none": "None",
  "gui.all": "All"
```

**Comment by ichttt** [comment]
Whoops, this broke when I started stripping comments... It's an easy fix, will patch it soon

**Comment by ichttt** [comment]
resolved by 56786fad662f2cd2df46e894bb1754df50d2a40d
